src/dali/plugin/pair_converter/ccxt.py

```diff
@@ -230,8 +253,8 @@ def exchange_markets(self) -> Dict[str, Dict[str, List[str]]]:
         return self.__exchange_markets
 
     @property
-    def exchange_graphs(self) -> Dict[str, MappedGraph[str]]:
-        return self.__exchange_graphs
+    def exchange_datetime_graph_tree_dict(self) -> Dict[str, AVLTree[datetime, MappedGraph[str]]]:
```

Comment:
Could you rename the function something like `get_graph_tree_from_exchange` or similar?

Reply:
How about `exchange_2_graph_tree`? That would be in line with the other properties. 

It only seems to be used for testing actually, it's not really necessary.

Reply:
Sure, sounds good.